src/condor_shadow.V6.1/baseshadow.cpp

```diff
@@ -137,8 +137,14 @@ BaseShadow::baseInit( ClassAd *job_ad, const char* schedd_addr, const char *xfer
 		prev_run_bytes_recvd = 0;
 	}
 
-	jobAd->LookupInteger(ATTR_TRANSFER_INPUT_FILES_TOTAL_COUNT, m_prev_run_upload_file_cnt);
-	jobAd->LookupInteger(ATTR_TRANSFER_OUTPUT_FILES_TOTAL_COUNT, m_prev_run_download_file_cnt);
+	ClassAd* prev_upload_stats = (ClassAd*)jobAd->Lookup(ATTR_TRANSFER_INPUT_STATS);
```

Comment:
Dynamic cast is safer, just in case the attribute value has a different type.
```suggestion
	ClassAd* prev_upload_stats = dynamic_cast<ClassAd*>(jobAd->Lookup(ATTR_TRANSFER_INPUT_STATS));
```